tasks/copy_from_archive/copy_from_archive.py

```diff
@@ -222,14 +226,15 @@ def copy_object(
         )
         LOGGER.debug(f"Object {src_object_name} copied.")
     except ClientError as ex:
-        LOGGER.error("Client error: {ex}", ex=ex)
+        LOGGER.error("Client error: {ex}")
         return ex.__str__()
     return None
 
 
 # noinspection PyUnusedLocal
+@LOGGER.inject_lambda_context(log_event=True)
```

Comment:
Is it possible to verify this call, at least in handler_happy_path tests?

Reply:
should be possible although I haven't tested this. Will check

Reply:
decorator has been removed from all lambdas.